lib/drives/pages/drive_detail_page.dart

```diff
@@ -354,29 +356,21 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
               child: Padding(
                 padding: const EdgeInsets.symmetric(horizontal: 5.0, vertical: 10.0),
                 child: Row(
-                  mainAxisAlignment: MainAxisAlignment.spaceAround,
                   children: <Widget>[
-                    Expanded(
-                      child: Align(
-                        alignment: Alignment.centerLeft,
-                        child: IconWidget(icon: icon, count: action.ride.seats),
-                      ),
-                    ),
-                    ProfileWidget(profile, size: 15, isTappable: false),
-                    Expanded(
-                      child: Align(
-                        alignment: Alignment.centerRight,
-                        child: badges.Badge(
-                          badgeContent: Text(
-                            action.ride.chat!.getUnreadMessagesCount().toString(),
-                            style: const TextStyle(color: Colors.white),
-                            textScaleFactor: 1.0,
-                          ),
-                          showBadge: action.ride.chat!.getUnreadMessagesCount() != 0,
-                          position: badges.BadgePosition.topEnd(top: -12),
-                          child: const Icon(Icons.chat),
-                        ),
+                    IconWidget(icon: icon, count: action.ride.seats),
+                    const SizedBox(width: 20),
+                    // TODO: Center this again
```

Comment:
@Eddie-42 I might need your help here. I remember we spent a long time trying to get this to the center. Now I had to rip that open again because the ProfileWidget itself needs to be Flexible.

Now it looks like this: (left-aligned but with the padding of 20 pixels).
![image](https://user-images.githubusercontent.com/63241108/221027463-558e9d0b-586f-43a4-ad7f-b1ae8d6a2f01.png)
But I could not figure out how to center it inside of the Flexible/Expanded. Do we want to leave it like this because that page in particular will be redesigned anyways?